specification.md

```diff
@@ -3673,17 +3674,17 @@ Service invocation can be done in two different ways:
 * Reference [functions definitions](#Function-Definition) by its unique name using the `functionRef` property.
 * Reference a `produced` and `consumed` [event definitions](#Event-Definition) via the `eventRef` property.
 
-In the event-based scenario a service or a set of services we want to invoke
+In some scenarios a service or a set of services which need to be invoked
 are not exposed via a specific resource URI for example, but can only be invoked via events.
 The [eventRef](#EventRef-Definition) defines the
 referenced `produced` event via its `triggerEventRef` property and a `consumed` event via its `resultEventRef` property.
 
-The `timeout` property defines the amount of time to wait for function execution to complete, or the consumed event referenced by the
-`resultEventRef` to become available.
-It is described in ISO 8601 format, so for example "PT2M" would mean the maximum time for the function to complete
-its execution is two minutes.
+The `sleep` property can be used to define time periods that workflow execution should sleep 
+before and/or  after function execution. It can have two properties:
+* `before` -  defines the amount of time (ISO 8601 duration format) to sleep before function invocation.
+* `after` -  defines the amount of time (ISO 8601 duration format) to sleep after function invocation.
```

Comment:
```suggestion
The `sleep` property can be used to define time periods that workflow execution should sleep
before and/or after function execution. It can have two properties:
* `before` -  defines the amount of time (ISO 8601 duration format) to sleep before function invocation.
* `after` -  defines the amount of time (ISO 8601 duration format) to sleep after function invocation.
```